NineChronicles.Headless/Properties/GraphQLNodeServiceProperties.cs

```diff
@@ -1,3 +1,5 @@
+using Org.BouncyCastle.Asn1.X509;
```

Comment:
It also seems unnecessary

Reply:
Yes, I amended them because they are unnecessary. 🙏🏻